helix-core/src/lib.rs

```diff
@@ -18,16 +18,8 @@ pub mod syntax;
 mod transaction;
 pub mod words;
 
-pub(crate) fn find_first_non_whitespace_char2(line: RopeSlice) -> Option<usize> {
-    // find first non-whitespace char
-    for (start, ch) in line.chars().enumerate() {
-        // TODO: could use memchr with chunks?
-        if ch != ' ' && ch != '\t' && ch != '\n' {
-            return Some(start);
-        }
-    }
-
-    None
+pub fn find_first_non_whitespace_char2(line: RopeSlice) -> Option<usize> {
+    line.chars().position(|ch| !ch.is_whitespace())
 }
 pub(crate) fn find_first_non_whitespace_char(text: RopeSlice, line_num: usize) -> Option<usize> {
```

Comment:
Can you get rid of this other function and then rename the `char2` one? It's used in a single place in indent.rs

Reply:
Done.